ivadomed/utils.py

```diff
@@ -276,8 +276,7 @@ def structurewise_uncertainty(fname_lst, fname_hard, fname_unc_vox, fname_out):
         elif len(data_i_l.shape) == 3:
             data_i_l = np.transpose(data_i_l, (1, 2, 0))
         else:
-            print("object does not appear to be 2D or 3D, please check the network output")
```

Comment:
why do you need to check this again-- doesn't it originate from the same variable as above?

Reply:
data_l_lst stems from "data" that comes from images in "fname_lst".  (see [here](https://github.com/ivadomed/ivadomed/blob/65037dffb4361a0686e02ec753caba4d095e39e3/ivadomed/utils.py#L241) , [here](https://github.com/ivadomed/ivadomed/blob/65037dffb4361a0686e02ec753caba4d095e39e3/ivadomed/utils.py#L249))
data_i_l from "data_hard" that comes from images in "fame_hard".  (see [here](https://github.com/ivadomed/ivadomed/blob/65037dffb4361a0686e02ec753caba4d095e39e3/ivadomed/utils.py#L270), and [here](https://github.com/ivadomed/ivadomed/blob/65037dffb4361a0686e02ec753caba4d095e39e3/ivadomed/utils.py#L221)).
It is linked but it is not the same files so I think it is better to check both.

Reply:
right-- but i would still check them at the beginning of the code-- also see my other comment about the check dim discrepancy

Reply:
I can check them for the erorr right but would still need an if statement here (to make the right transpose)